handlers/lobbies.js

```diff
@@ -7,34 +7,187 @@ Dota2.SeriesType = {
     BEST_OF_FIVE: 2
 };
 
+Dota2.BotDifficulty = {
+    PASSIVE: 0,
+    EASY: 1,
+    MEDIUM: 2,
+    HARD: 3,
+    UNFAIR: 4
+};
+
 Dota2._lobbyOptions = {
     game_name: "string",
+    pass_key: "string",
     server_region: "number",
     game_mode: "number",
     game_version: "number",
     cm_pick: "number",
     allow_cheats: "boolean",
     fill_with_bots: "boolean",
+    bot_difficulty_radiant: "number",
+    bot_difficulty_dire: "number",
+    bot_radiant: "number",
+    bot_dire: "number",
     allow_spectating: "boolean",
-    pass_key: "string",
     series_type: "number",
     radiant_series_wins: "number",
     dire_series_wins: "number",
+    previous_match_override: "number",
     allchat: "boolean",
-    leagueid: "number",
     dota_tv_delay: "number",
+    leagueid: "number",
     custom_game_mode: "string",
     custom_map_name: "string",
     custom_difficulty: "number",
     custom_game_id: "number",
 };
 
 // Methods
-Dota2.Dota2Client.prototype.createPracticeLobby = function(password, options, callback) {
+
+/**
+ * Sends a message to the Game Coordinator requesting to create a lobby. Listen for
+ * practiceLobbyUpdate response for a snapshot-update of the newly created lobby.
+ *
```

Comment:
Add `@alias module:Dota2.Dota2Client#createPracticeLobby` and change practiceLobbyUpdate to `{@link module:Dota2.Dota2Client#event:practiceLobbyUpdate|practiceLobbyUpdate}`